packages/types/src/types/session/database-interface.ts

```diff
@@ -21,5 +21,5 @@ export interface DatabaseInterfaceSessions {
 
   invalidateSession(sessionId: string): Promise<void>;
 
-  invalidateAllSessions(userId: string): Promise<void>;
+  invalidateAllSessions(userId: string, excludeList?: string[]): Promise<void>;
```

Comment:
What do you think about having a self-explanatory name instead of something generic (eg: excludeSessionIds or smth like this)?

Reply:
Yup. make sense. 👍